src/main/studio/ui/EditorPane.java

```diff
@@ -96,18 +97,35 @@ public RSyntaxTextArea getTextArea() {
         return textArea;
     }
 
-    public void setStatus(String status) {
+    public void setStatus(String status, boolean isAlert) {
         lblStatus.setText(status);
+        if (isAlert) {
+            lblStatus.setForeground(Color.WHITE);
+            boxStatus.setOpaque(true);
+            boxStatus.setBackground(Color.RED);
```

Comment:
Showing errors with a different style (bringing attention) is a good thing. However my view that red background is too contrast. My view that even just red font brings attentions

Reply:
I can try that, this choice was only based on what the NetBeans editor was doing which was to show white text on red background.